### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by luisdavim** [garabik/grc#149]
Any plans to merge devel into master and having a look at the open PRs?

**Comment by garabik** [comment]
Yes, I still maintain `grc`, and I am planning a new release soon (though life interfered, unfortunately...)

**Comment by luisdavim** [comment]
cool, let me know if you need help.

**Comment by mjustin** [comment]
A new release would be great.  I use the [Homebrew](https://brew.sh/) package manager on macOS to install `grc`, and it would be nice to be able use the last 2+ years of updates.

**Comment by jamieweavis** [comment]
I'm also using `grc` with Homebrew and would really appreciate a new release to take advantage of the extra ZSH commands added in #152 🤞  (along with the other features and fixes added since the `1.11.3` release ofcourse!)

**Comment by otherguy** [comment]
Same here!

**Comment by garabik** [comment]
Yes, it is maintained, unfortunately current situation interfered with my intentions... Nevertheless, I hope to make a release soon.
One point is that I do not use zsh, nor Mac, so improvements in these areas are completely upon external contributions and I cannot do much (or at all) testing...

**Comment by otherguy** [comment]
@garabik thank you for your answer!

I happen to use `zsh` on macOS, so if you want me to do some testing for you, feel free to let me know. I'd be happy to contribute.

**Comment by garabik** [comment]
Just keep using the `devel` branch and report any bugs you find... (actually, I plan to merge the `devel` and `master` branches, it was a hassle to maintain and somewhat unnecessary)

**Comment by luisdavim** [comment]
when can we expect `devel` to be merged into master and a new release?

**Comment by garabik** [comment]
> when can we expect `devel` to be merged into master and a new release?

Well, I wanted to do it sooner but real life issues prevented it...I've made a release just now.

**Comment by mitchblank** [comment]
> Well, I wanted to do it sooner but real life issues prevented it...I've made a release just now.

@garabik -- the release doesn't seem to have been tagged correctly.  I can see that "master" now has the 1.12 bits but if you look at https://github.com/garabik/grc/releases you'll see that both v.11.3 and v.12 are at the same ref (92aac17)    I just downloaded the v1.12 tarball (as part of upgrading Homebrew) and found it had the v1.11.3 bits inside!
```
$ grep 'Generic Colouriser' grc-1.12/grc
    print ("Generic Colouriser 1.11.3")
    print("""Generic Colouriser 1.11.3
```

**Comment by garabik** [comment]
> @garabik -- the release doesn't seem to have been tagged correctly. I can see that "master" now has the 1.12 bits but if you look at https://github.com/garabik/grc/releases you'll see that both v.11.3 and v.12 are at the same ref ([92aac17]

Thanks for noticing, it should be fixed now.

**Comment by mitchblank** [comment]
Yep looks good to me now.

Opened https://github.com/Homebrew/homebrew-core/pull/72226, we'll see how it goes.

**Comment by HaleTom** [comment]
Can this issue be closed now?    An old issue with this title could well give the wrong impression :)

**Comment by krodelabestiole** [comment]
What impression ?
:eyes:

(I have just discovered this tool and I find it great, so I hope for the best.)

**Comment by HaleTom** [comment]
It could well appear that the project hasn't been maintained for 4 years.

**Comment by luisdavim** [comment]
Well, the last commit wad 2 years ago...

**Comment by HaleTom** [comment]
There may be some good updates [here](https://github.com/garabik/grc/compare/master...kriipke:grc:master) and [here](https://github.com/garabik/grc/compare/master...adamhotep:grc:master) to pull in from the last couple of years.

**Comment by huyz** [comment]
Anyone have a good fork that pulls in good contributions?
